t/030-uri-args.t

```diff
@@ -1298,8 +1312,8 @@ foo=3&a%20b=32&bar=5
     }
 --- request
 GET /lua
---- response_body
-foo=3&a%20b=32&a%20b&bar=5
+--- response_body_like
+^(?:a%20b(?:&(?:bar=5&(?:a%20b=32&foo=3|foo=3&a%20b=32)|foo=3&(?:a%20b=32&bar=5|bar=5&a%20b=32)|a%20b=32&(?:bar=5&foo=3|foo=3&bar=5))|=32&(?:a%20b&(?:bar=5&foo=3|foo=3&bar=5)|bar=5&(?:a%20b&foo=3|foo=3&a%20b)|foo=3&(?:a%20b&bar=5|bar=5&a%20b)))|bar=5&(?:a%20b(?:&(?:a%20b=32&foo=3|foo=3&a%20b=32)|=32&(?:a%20b&foo=3|foo=3&a%20b))|foo=3&a%20b(?:&a%20b=32|=32&a%20b))|foo=3&(?:a%20b(?:&(?:a%20b=32&bar=5|bar=5&a%20b=32)|=32&(?:a%20b&bar=5|bar=5&a%20b))|bar=5&a%20b(?:&a%20b=32|=32&a%20b)))$
```

Comment:
Yes this is much better way. Easy to understand and quite simple to update.

& test should have $ at the end.